infra/abbreviations.json

```diff
@@ -131,5 +131,6 @@
   "webSitesAppService": "app-",
   "webSitesAppServiceEnvironment": "ase-",
   "webSitesFunctions": "func-",
-  "webStaticSites": "stapp-"
+  "webStaticSites": "stapp-",
+  "searchIndexName":"gptkbindex"
```

Comment:
The Abbreviations file seemed to be the place where default values were pulled from. This matches the other items (example Main.bicep line 92, the resource token is not needed here because the index is not required to be globally unique)

Reply:
This abbreviations files came from the azd repository (https://github.com/Azure/azure-dev/blob/main/templates/common/infra/bicep/abbreviations.json) and stores default prefixes for created Azure resources, so that all azd templates create resources with similar naming scheme. The index name is more akin to the model deployment name, like "davinci", which aren't stored in here.

Reply:
Understood. I am running a test on this now to test before I resolve the comment